.env.development

```
@@ -0,0 +1,15 @@
+VITE_USE_MOCK_AUTH=true
+VITE_FIREBASE_API_KEY=AIzaSyClozThfUdOOXnC0gkHz_DCVP1t75B5hro
+VITE_FIREBASE_AUTH_DOMAIN=gmshooter.firebaseapp.com
+VITE_FIREBASE_PROJECT_ID=gmshooter
+VITE_FIREBASE_STORAGE_BUCKET=gmshooter.firebasestorage.app
+VITE_FIREBASE_MESSAGING_SENDER_ID=97079543510
+VITE_FIREBASE_APP_ID=1:97079543510:web:4db510e1cbfc6256546b77
+VITE_FIREBASE_MEASUREMENT_ID=mock-measurement-id
+VITE_ROBOFLOW_API_KEY=mock-roboflow-key
+VITE_ROBOFLOW_MODEL_ID=mock-model-id
+VITE_NGROK_URL=http://localhost:4040
+VITE_GEMINI_API_KEY=AIzaSyC3AhQbkUSiJm2ZvuNyC0MNqv3I0hFE0Sk
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_GEMINI_API_KEY key should go before the VITE_NGROK_URL key

(UnorderedKey)